src/utils/supabase/client.test.ts

```
@@ -0,0 +1,333 @@
+      createClient();
+
+      expect(createBrowserClient).toHaveBeenCalledWith(
+        process.env.NEXT_PUBLIC_SUPABASE_URL,
+        process.env.NEXT_PUBLIC_SUPABASE_ANON_KEY
+      );
+    });
+  });
+
+  describe('createClient - Environment Variable Edge Cases', () => {
+    test('should throw error when NEXT_PUBLIC_SUPABASE_URL is undefined', () => {
+      delete process.env.NEXT_PUBLIC_SUPABASE_URL;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)